src/vue/common/OperationRecord.vue

```diff
@@ -0,0 +1,215 @@
+<template>
```

Comment:
OperationRecord.vue doesn’t describe what’s going on actually. I’d recommend renaming it